components/ActivityCard/index.js

```diff
@@ -1,27 +1,21 @@
 import Image from "next/image";
 import styled from "styled-components";
+import Link from "next/link";
+import StyledImageComponent from "../StyledImageComponent";
 
 export default function ActivityCard({ activity }) {
   return (
     <article>
-      <ImageContainer>
-        <StyledImage src={activity.image} alt={activity.title} fill />
-      </ImageContainer>
-
-      <h2>{activity.title}</h2>
-      <span>
-        {activity.area}, {activity.country}
-      </span>
-      <div>{activity.category.join(", ")}</div>
-      <p>{activity.description}</p>
+      <Link href={`/${activity.id}`}>
+        <StyledImageComponent src={activity.image} alt={activity.title} />
+        <h2>{activity.title}</h2>
+      </Link>
+      <div>
```

Comment:
What is the purpose of this `<div>`?

Reply:
We thought it would be easier to style later, but it's not really needed now